config/routes.rb

```diff
@@ -3,11 +3,17 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'start#index'
-
+  root 'reports#new'
   get 'cookies' => 'start#cookies'
-  get 'select_soc_codes' => 'start#select_soc_codes'
-  get 'report' => 'start#report'
+
+  resources :reports, :path => '/' do
+    member do
+      get 'select_soc_codes'
+      get 'show'
+      patch 'save_soc_codes'
```

Comment:
`get 'soc_codes', to: :select_soc_codes` and `patch 'soc_codes', to: :save_code_codes` would be less RPC-ish (which I think is better for an HTTP endpoint)